pandas/tests/frame/indexing/test_setitem.py

```diff
@@ -229,3 +229,10 @@ def test_setitem_periodindex(self):
         rs = df.reset_index().set_index("index")
         assert isinstance(rs.index, PeriodIndex)
         tm.assert_index_equal(rs.index, rng)
+
+    def test_iloc_setitem_bool_array(self):
```

Comment:
this is actually a list can you parameterize on 
```[list, np.array]```  e.g.
```
indexer = klass([True, False, False])
df.iloc[indexer, 1] = .....
```

Reply:
Good point. Done. Renamed test too.